zstacklib/zstacklib/utils/multipath.py

```
@@ -78,21 +102,35 @@ def write_multipath_conf(path, blacklist=None):
 
-        logger.info(config)
-        if modified:
+        self.modified |= modified
+        return modified
+
+    def config_section(self, name, cfg):
+        for section in self.config:
+            if name not in section:
+                continue
+
+            if cmp(sorted_conf(section[name]), sorted_conf(cfg)) != 0:
```

Ruff (0.12.2)

Undefined name `cmp`

(F821)